pkg/granted/request/close.go

```diff
@@ -0,0 +1,150 @@
+package request
+
+import (
+	"context"
+	"fmt"
+
+	"connectrpc.com/connect"
+	"github.com/common-fate/cli/printdiags"
+	"github.com/common-fate/clio"
+	"github.com/common-fate/grab"
+	"github.com/common-fate/granted/pkg/cfaws"
+	"github.com/common-fate/granted/pkg/cfcfg"
+	"github.com/common-fate/sdk/config"
+	"github.com/common-fate/sdk/eid"
+	accessv1alpha1 "github.com/common-fate/sdk/gen/commonfate/access/v1alpha1"
+	"github.com/common-fate/sdk/service/access/grants"
+	"github.com/common-fate/sdk/service/access/request"
+	identitysvc "github.com/common-fate/sdk/service/identity"
+	"github.com/urfave/cli/v2"
+)
+
+var closeCommand = cli.Command{
+	Name:  "close",
+	Usage: "Close an active Just-In-Time access to a particular entitlement",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "aws-profile", Required: false, Usage: "Close a JIT access for a particular AWS profile"},
+		&cli.StringFlag{Name: "request-id", Required: false, Usage: "Close a JIT access for a particular access request ID"},
+	},
+	Action: func(c *cli.Context) error {
+		profiles, err := cfaws.LoadProfiles()
```

Comment:
Nit: you could shift this `profiles, err := cfaws.LoadProfiles()` to only load if the profile flag is being used, put it inside the if

Lets add a validation warning indicating the behaviour if both flags are provided

```
if c.String("request-id") != "" && c.String("profile") != "" {
clio.Warn("Both profile and request-id were provided, profile will be ignored")
}
```